PagingSample/app/build.gradle

```diff
@@ -50,25 +50,29 @@ android {
         jvmTarget = "1.8"
         freeCompilerArgs += ["-Xopt-in=kotlin.RequiresOptIn"]
     }
+
```

Comment:
what are these changes for?

Reply:
This is a competing kotlin / coroutine versions thing where the same license is getting added multiple times :|

Reply:
Yeah I'm really not sure, but it seems to fail consistent in circle ci without it.